examples/research_projects/wav2vec2/vocab/buckwalter.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Can you give more context regarding this file? Why do we add it here? Also should tokens like "}" really be included in the vocab? How would this vocab be used for languages that have special tokens, like Chinese, Japanese, ...? I'm wondering whether this might be more confusing that helpful

Reply:
I want to pass the vocab dict in python, what's the best way (besides changing `Wav2Vec2CTCTokenizer`)? I had an iteration with `add_tokens` but I figured I should remove tokens as well to match [Buckwalter orthography](http://www.qamus.org/transliteration.htm). It's a way to express Arabic in ASCII, which is what the [arabic_speech_corpus](https://huggingface.co/datasets/arabic_speech_corpus#data-instances) dataset uses.
I figure Japanese uses ASCII characters that also happen to be alpha: https://github.com/kariminf/lang-trans/blob/a957a3fa549e674d20e94e30935e028994673067/lang_trans/japanese.py

Reply:
I also explain this in `README.md` after your suggestion.

Reply:
Gottcha! Ok that's fine for me!